cms/djangoapps/contentstore/views/course.py

```diff
@@ -1459,32 +1459,109 @@ def _get_usage_info(store, course, split_tests):
+            [
+                {
+                    'label': 'Unit 1 / Problem 1',
+                    'url': 'url_to_unit_1'
+                },
+                {
+                    'label': 'Unit 2 / Problem 2',
+                    'url': 'url_to_unit_2'
+                }
+            ],
+        }
+        """
+        usage_info = {}
+        for module in modules:
+            if hasattr(module, 'group_access') and module.group_access:
+                (__, group_ids), = module.group_access.items()
+                for group_id in group_ids:
+                    if group_id not in usage_info:
+                        usage_info[group_id] = []
+
+                    unit_location = store.get_parent_location(module.location)
+                    if not unit_location:
+                        log.warning("Parent location of content_groups module not found: %s", module.location)
+                        continue
+
+                    try:
+                        unit = store.get_item(unit_location)
+                    except ItemNotFoundError:
+                        log.warning("Unit not found: %s", unit_location)
+                        continue
+
+                    unit_url = reverse_usage_url(
+                        'container_handler',
+                        course.location.course_key.make_usage_key(unit.location.block_type, unit.location.name)
+                    )
+
+                    usage_info[group_id].append({
+                        'label': u"{} / {}".format(unit.display_name, module.display_name),
+                        'url': unit_url,
+                    })
+        return usage_info
+
+    @staticmethod
     def update_usage_info(store, course, configuration):
         """
         Update usage information for particular Group Configuration.
 
         Returns json of particular group configuration updated with usage information.
         """
-        # Get all Experiments that use particular Group Configuration in course.
-        split_tests = store.get_items(
-            course.id,
-            category='split_test',
-            content={'user_partition_id': configuration.id}
-        )
-        configuration_json = configuration.to_json()
-        usage_information = GroupConfiguration._get_usage_info(store, course, split_tests)
-        configuration_json['usage'] = usage_information.get(configuration.id, [])
+        # Get all Experiments that use particular  Group Configuration in course.
+        if configuration.scheme.name == 'random':
+            split_tests = store.get_items(
+                course.id,
+                category='split_test',
+                content={'user_partition_id': configuration.id}
+            )
+            configuration_json = configuration.to_json()
+            usage_information = GroupConfiguration._get_usage_info(store, course, split_tests)
+            configuration_json['usage'] = usage_information.get(configuration.id, [])
+        else:
```

Comment:
I would go ahead and check specifically for scheme.name == "cohort", as we anticipate having other schemes in the future.